session.go

```diff
@@ -91,6 +93,52 @@ func NewSession(stdout, stderr io.Writer) (*Session, error) {
 	return s, nil
 }
 
+func (s *Session) initGoMod() (err error) {
+	pwd, module, err := getCurrentModule()
+	if err != nil || module == "" {
+		return
+	}
+
+	s.useGoMod = true
+
+	tempModule := filepath.Base(s.tempDir)
+	goModPath := filepath.Join(s.tempDir, "go.mod")
+
+	mod := fmt.Sprintf("module %s\nrequire %s v0.0.0\nreplace %s => %s\n", tempModule, module, module, pwd)
```

Comment:
with the `require` and `replace` magic, we could import package from current project having go.mod

Reply:
Good idea used `replace` to build temporary module for console setting